Python/Product/PythonTools/PythonTools/Project/PythonProjectNode.cs

```diff
@@ -1725,6 +1814,8 @@ private static T FindImmediateChild<T>(HierarchyNode parent)
                         return "e,env,environment: p,package: a,admin";
                     case PythonConstants.GenerateRequirementsTxt:
                         return "e,env,environment:";
+                    case PythonConstants.ProcessRequirementsTxt:
+                        return "path";
```

Comment:
Just try invoking the command and passing the arguments. Though it may be irrelevant unless you're using [IVsParseCommandLine](https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.shell.interop.ivsparsecommandline.aspx) (like we do for the other commands, via the `ParseCommandArgs` helper)